src/utils/ExecsList.js

```diff
@@ -0,0 +1,160 @@
+import praneer from '../assets/execs/praneer.svg'
+import grace from '../assets/execs/grace.svg'
+import noble from '../assets/execs/noble.svg'
+import vivian from '../assets/execs/vivian.svg'
+import naomi from '../assets/execs/naomi.svg'
+import isshan from '../assets/execs/isshan.svg'
+import david from '../assets/execs/david.svg'
+import revika from '../assets/execs/revika.svg'
+import nargis from '../assets/execs/nargis.svg'
+import harry from '../assets/execs/harry.svg'
+import shubham from '../assets/execs/shubham.svg'
+import emma from '../assets/execs/emma.svg'
+import ali from '../assets/execs/ali.svg'
+import fiona from '../assets/execs/fiona.svg'
+import steven from '../assets/execs/steven.svg'
+import jocelyn from '../assets/execs/jocelyn.svg'
+import michelle from '../assets/execs/michelle.svg'
+import lauren from '../assets/execs/lauren.svg'
+import nathan from '../assets/execs/nathan.svg'
+import karren from '../assets/execs/karren.svg'
+import josh from '../assets/execs/josh.svg'
```

Comment:
instead of importing each image, you can import an entire folder like so:
```
const resolver = require.context('../assets/execs', false, /.*\.png/)
```

then in a `src={ ... }` you can put `resolver('image.png')`

so for example, below instead of storing the imported sources, instead store the name of the image like 'josh.svg', then in your map in the main component you can do something like this

```
[near the top w/ your other imports]
const execResolver = require.context('../assets/execs', false, /.*\.png/)


[at the map]
execs.map(exec => <img src={execResolver(exec.src)} />)
```